packages/web-components/package.json

```diff
@@ -219,6 +219,7 @@
     "@microsoft/fast-element": "2.0.0-beta.26",
     "@types/web": "^0.0.142",
     "@storybook/html": "6.5.15",
+    "@tensile-perf/tools": "0.1.7",
```

Comment:
can we move this to root package.json root to follow [SVP](https://github.com/microsoft/fluentui/issues/21395) for devDeps  @spmonahan @chrisdholt  ?

Reply:
Sounds good to me but I defer to Chris.

Reply:
I'm good with this - I tried it in the initial PR and I the build blew up with some wild errors...my assumption is there is something using chromedriver perhaps that I'm not aware of.

Reply:
I moved the 2 tensile-perf packages, anything I should move?

Reply:
I think we should do this as a separate PR IMO - it seems out of scope given you just moved it to be alphabetized (it already existed in place I believe).

Reply:
I can revert the change to `@tensile-perf/web-components`, but I should keep the addition of `@tensile-perf/tools` in the root package.json right?

Reply:
Any change to this should be a separate PR IMO. I made the change in the PR which updated this initially and ran into a lot of breaks which led me to believe that the move may have *other* changes required. I'd prefer that we not make the dependency change at all as part of this PR as it's a separate concern.

Reply:
The `set-theme.bench.ts` file does need `@tensile-perf/tools` so I have to add it to either the root or w-c's package.json.

However, if `@tensile-perf/web-components` package had exported the `measurePerformance()` function, I wouldn't need `@tensile-perf/tools` at all.

For now, I can move `@tensile-perf/tools` back to w-c's package.json and optimize the dependencies in a separate PR. What do you think?

Reply:
It exists today in the package.json (before any changes to the branch) - https://github.com/microsoft/fluentui/blob/1e1398cb66c56c6f67c33c33a54949cd24190a35/packages/web-components/package.json#L222

Reply:
> The `set-theme.bench.ts` file does need `@tensile-perf/tools` so I have to add it to either the root or w-c's package.json.
> 
> However, if `@tensile-perf/web-components` package had exported the `measurePerformance()` function, I wouldn't need `@tensile-perf/tools` at all.
> 
> For now, I can move `@tensile-perf/tools` back to w-c's package.json and optimize the dependencies in a separate PR. What do you think?

@spmonahan can you take a look here and advise if we _need_ the tools package or if this is internalized as part of the benchmarks as is?

Reply:
I think I'll update the `/web-components` to re-export the relevent APIs from `/tools` since that package already depends on `/tools` and users should only really be concerned with their specific platform's package (web-components, React).

Reply:
@chrisdholt and @marchbox This should take care of you :)

https://github.com/microsoft/tensile-perf/pull/32

Reply:
[v0.2.0](https://www.npmjs.com/package/@tensile-perf/web-components/v/0.2.0) should have the exports you need.

Reply:
Thanks @spmonahan! I've removed `@tensile-perf/tools` from dependencies and upgraded `@tensile-perf/web-components` to `~0.2.0`.